.travis.yml

```diff
@@ -52,6 +52,10 @@ jobs:
     # define a separate script for each "examples/*" folder
     # this will run it in a separate job on TravisCI
     # all these jobs belong to same stage "test" thus will run in parallel
+    - stage: test
+      env:
+        - CYPRESS_framework=kotlin-react
```

Comment:
Frameworks are arranged in alphabetical order

Reply:
done..